### PR TITLE
feat: added course-search

### PR DESCRIPTION
**Pull request by burhanuday**
course_search_1

here is the repository
https://github.com/burhanuday/course-search

### CODE REVIEW

**Comment by HashimotoYT** [comment]
ping @burhanuday

**Comment by burhanuday** [comment]
@HashimotoYT 
I have made the necessary changes

File: apps/course-search/course-search.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+name: 'Course Search'
+description: 'Find courses from multiple sources available to download in torrent form.'
+website: 'https://github.com/burhanuday/course-search'
+repository: 'https://github.com/burhanuday/course-search'
+keywords:
+    - Course Search
+    - 'cross platform'
```
Comment (Sanjit1) [inline apps/course-search/course-search.yml]: Remove the quotes, and make it just `cross platform`

**Comment by Sanjit1** [comment]
ping @HashimotoYT 
Please review the changes

**Comment by burhanuday** [comment]
@HashimotoYT please review

**Comment by burhanuday** [comment]
Hello, are there any other changes required?

**Comment by Toinane** [comment]
Hey there, it seems that the application is no longer maintained and has not been updated for 12 months. 
So I'm going to close this PR for now. Feel free to reopen it or make a new PR if needed

**Pull request closed without merging**